core/src/main/scala/org/bitcoins/core/crypto/BIP39Seed.scala

```diff
@@ -58,4 +58,18 @@ object BIP39Seed extends Factory[BIP39Seed] {
     BIP39Seed.fromBytes(ByteVector(encodedBytes))
   }
 
+  /**
+    * Generates a [[https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki BIP32]]
+    * seed from a mnemonic code. An optional password can be supplied.
+    */
+  def fromMnemonic(
```

Comment:
Do we need the other `fromMnemonic()` defined in this object?

Reply:
There already is another